src/components/network-chart/NetworkChartRow.js

```diff
@@ -14,8 +14,12 @@ import {
   getColorClassNameForMimeType,
```

Comment:
Tthis commit should be easier to review by setting the "hide whitespace changes" parameter. It's mostly moving code around.